docs/Catalogs.adoc

```diff
@@ -0,0 +1,277 @@
+= Using Operator Catalog Snapshots
+
+== Overview
+
+Operators in OpenShift change quite frequently - even within operator subscription channels. Also the available ClusterServiceVersions for any given operator change very frequently with older versions being removed from the catalog - even though the container images making up the operator and/or managed applications still exist in the various container registries.
+
+In order to have a predictable environment for demos or classroom environments it is preferable to install exactly the operator version that the demo / lab was written for.
+
+One way to achieve this is by creating an operator catalog snapshot. There are 4 catalogs in OpenShift (as of July 2020):
+
+* Red Hat Operators
+* Certified Operators
+* Community Operators
+* Marketplace Operators
+
+Creating an operator catalog snapshot means creating a container image containing the YAML definitions of a catalog at a state in time. Once the catalog image has been built it needs to be hosted in a (public) container registry. Then a catalog source  can be created that points to this particular catalog image which will no longer change - even if the online catalogs get updated with newer releases of any given operator.
+
+The process for creating an operator catalog snapshot image is:
```

Comment:
I'm not sure what change you pushed, but this is the section I was referring to.

You are saying "here is the process to create an operator catalog snapshot image"

But the steps you outline include that as well as deploying an actual operator. It would make more sense to break this up and make it clear that there are certain steps to create the image and certain steps to deploy from that image.

Reply:
I honestly don't get what's not clear here. You create a snapshot. You host the snapshot image in a registry. You create a catalog source pointing to the registry.

This is not hard.